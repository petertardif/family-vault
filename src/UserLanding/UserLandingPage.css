.userlandingpage-header {
  padding-top: 150px;
  /* min-height: 50px; */
}