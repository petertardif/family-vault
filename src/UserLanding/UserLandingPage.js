import React, { Component } from 'react';
import { withRouter } from 'react-router-dom';
import './UserLandingPage.css';

class UserLandingPage extends Component {

handleClickAddMemory = () => {
  this.props.history.push('/add-memory');
}

handleClickGoBack = () => {
  this.props.history.push('/memorylist');
}

  render() {
    return (
      <div className='userlandingpage-header'>
        <header >
          <h1>Create and forever cherish the moments!</h1>
        </header>
        <section className='userlandingpage-footer'>
          <button onClick={this.handleClickAddMemory} className='Button blue'>Add a Memory</button>
          <button onClick={this.handleClickGoBack} className='Button blue'>Memory Page</button>
        </section>
      </div>
    )
  }
}

export default withRouter(UserLandingPage);
