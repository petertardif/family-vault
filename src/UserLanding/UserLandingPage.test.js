import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import UserLandingPage from './UserLandingPage'

describe(`UserLandingPage component`, () => {

  it('renders a .UserLandingPage by default', () => {
    const wrapper = shallow(<UserLandingPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
