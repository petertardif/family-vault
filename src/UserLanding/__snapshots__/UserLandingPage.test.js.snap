// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserLandingPage component renders a .UserLandingPage by default 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
